JuliaInterface/gap/JuliaInterface.gi

```diff
@@ -57,11 +57,74 @@ InstallMethod( CallFuncList,
     end );
 
 
-##
-##  We support '<juliaobj>[<i>]' and '<juliaobj>[<i>, <j>]' in general.
-##  This is useful for example if <juliaobj> is a tuple.
-##
-BindJuliaFunc( "getindex", "Base" );
+BindGlobal( "_JULIA_MODULE_TYPE", JuliaEvalString( "Module" ) );
+BindGlobal( "_JULIA_FUNCTION_TYPE", JuliaEvalString( "Function" ) );
+BindGlobal( "_JULIA_ISA", JuliaFunction( "isa" ) );
+
+BindGlobal( "_NewJuliaModule",
```

Comment:
I find this name confusing: It refers to `IsJuliaModule` type objects, of course, but I keep reading it as "create a new Julia module", i.e., on the Julia side. So it is ambiguous. Perhaps: `_WrapJuliaModule`?